kubernetes-AKS/secret.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pricepointscout-secret
+  namespace: pps-namespace
+type: Opaque
+data:
+  DATABASE_PASSWORD: ""  # Base64 encoded password
+  JWT_SECRET: "bXktdWx0cmEtc2VjdXJlLWFuZC11bHRyYS1sb25nLXNlY3JldA=="  # Base64 encoded secret
+  EMAIL_USERNAME: "cHJpY2Vwb2ludHNjb3V0QGdtYWlsLmNvbQ=="  # Base64 encoded email
+  EMAIL_PASSWORD: "cHJpY2Vwb2ludHNjb3V0MTIz"  # Base64 encoded password
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)